app/status.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python2.7
```

Comment:
I think this is introducing a lot of duplicate code in the codebase. We should be either:
  - removing the duplicate code from create.py and make create.py import some shared functionality from this file, or vise verse, or
  - add some conditions to create.py so it can also handle status requests.

Also same as the above comment. Please rename this to add_note.py